mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -17,66 +20,42 @@ pipeline {
         stage('Build and Test') {
             steps {
                 checkout scm
-                sh '''
-                    mkdir -p build && cd build
-                    # Set the path to llvm-symbolizer
-                    PATH=$PATH:/opt/rocm-4.1.1/llvm/bin/
-
-                    cmake -G Ninja ../llvm \
-                        -DLLVM_ENABLE_PROJECTS="mlir;lld" \
-                        -DLLVM_BUILD_EXAMPLES=ON \
-                        -DLLVM_TARGETS_TO_BUILD="X86;AMDGPU" \
-                        -DCMAKE_BUILD_TYPE=Release \
-                        -DLLVM_ENABLE_ASSERTIONS=ON \
-                        -DBUILD_SHARED_LIBS=ON \
-                        -DLLVM_BUILD_LLVM_DYLIB=ON \
-                        -DMLIR_ROCM_RUNNER_ENABLED=1 \
-                        -DMLIR_MIOPEN_DRIVER_ENABLED=1 \
-                        -DMLIR_MIOPEN_DRIVER_PR_E2E_TEST_ENABLED=1 
-                        # if we open sqlite, step4 lowering failed until adding the custom db
-                        # -DMLIR_MIOPEN_SQLITE_ENABLED=1
-
-                    # Build LLVM / MLIR and run tests
-                    cmake --build . --target check-mlir
-                '''
+                cmakeBuild generator: 'Ninja',\
+                    buildDir: 'build',\
+                    buildType: 'Release',\
+                    installation: 'InSearchPath',\
+                    steps: [[args: 'check-mlir']],\
+                    cmakeArgs: '''-DMLIR_MIOPEN_DRIVER_ENABLED=1
+                                   -DMLIR_MIOPEN_DRIVER_PR_E2E_TEST_ENABLED=1
+                                   -DCMAKE_EXPORT_COMPILE_COMMANDS=1
```

Comment:
[Question] I believe the last configuration explicitly ask to export compile commands, but what about the rest of the configurations in the left side of the diff?

Reply:
1) It's possible we don't even need this line, since it looks like the root cause of the failure was me using `[[` instead of `[` in a `sh` block
2) The files that aren't `downstream` or `downstream.xdlop` don't run the static checks that need this file.

Reply:
What about all the other arguments in the original downstream file:

```cmake
-DCMAKE_BUILD_TYPE=Release \
-DLLVM_ENABLE_ASSERTIONS=ON \
-DBUILD_SHARED_LIBS=ON \
-DLLVM_BUILD_LLVM_DYLIB=ON \
-DMLIR_ROCM_RUNNER_ENABLED=1
```

Reply:
Those are handled via the fact that we're using the CMakeLists.txt in the root directory and not in ../llvm